docs-source/content/samples/simple/elastic-stack/weblogic-domain/_index.md

```diff
@@ -0,0 +1,335 @@
+
+Example: `kubectl edit domain bobs-bookstore -n bob` and add the following portions to the domain definition.  
+```yaml
+spec:
+  serverPod:
+    volumes:
+    - configMap:
+        defaultMode: 420
+        name: fluentd-config
+      name: fluentd-config-volume
+```
+
+#### Add fluentd Container
+Add a container to the domain that will run `fluentd` in the admin and managed server pods.
+
+Notice the container definition:
+* Defines a LOG_PATH environment variable points to the log location of `bobbys-front-end`
+* Defines ELASTICSEARCH_HOST, ELASTICSEARCH_PORT, ELASTICSEARCH_USER, and ELASTICSEARCH_PASSWORD environment variables that are all retrieving their value from the secret `bobs-bookstore-weblogic-credentials`.
+* Has volume mounts for the `fluentd-config` ConfigMap and the volume containing the domain logs
+
+NOTE: For brevity only the paths to the relevant configuration being added is shown.  A complete example of a domain definition is at the end of this document.
+
+Example: `kubectl edit domain bobs-bookstore -n bob` and add the following container definition.
+```yaml
+spec:
+  serverPod:
+    containers:
+    - args:
+      - -c
+      - /etc/fluent.conf
+      env:
+      - name: DOMAIN_UID
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.labels['weblogic.domainUID']
+      - name: SERVER_NAME
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.labels['weblogic.serverName']
+      - name: LOG_PATH
+        value: /scratch/logs/bobs-bookstore/$(SERVER_NAME).log
+      - name: FLUENTD_CONF
+        value: fluentd.conf
+      - name: FLUENT_ELASTICSEARCH_SED_DISABLE
+        value: "true"
+      - name: ELASTICSEARCH_HOST
+        valueFrom:
+          secretKeyRef:
+            key: elasticsearchhost
+            name: bobs-bookstore-weblogic-credentials
+      - name: ELASTICSEARCH_PORT
+        valueFrom:
+          secretKeyRef:
+            key: elasticsearchport
+            name: bobs-bookstore-weblogic-credentials
+      - name: ELASTICSEARCH_USER
+        valueFrom:
+          secretKeyRef:
+            key: elasticsearchuser
+            name: bobs-bookstore-weblogic-credentials
+            optional: true
+      - name: ELASTICSEARCH_PASSWORD
+        valueFrom:
+          secretKeyRef:
+            key: elasticsearchpassword
+            name: bobs-bookstore-weblogic-credentials
+            optional: true
+      image: fluent/fluentd-kubernetes-daemonset:v1.3.3-debian-elasticsearch-1.3
+      imagePullPolicy: IfNotPresent
+      name: fluentd
+      resources: {}
+      volumeMounts:
+      - mountPath: /fluentd/etc/fluentd.conf
+        name: fluentd-config-volume
+        subPath: fluentd.conf
+      - mountPath: /scratch
+        name: weblogic-domain-storage-volume
+```
+
+#### Verify Logs Exported to ELK
+
+Once the admin and managed server pods have started with all the changes describe above, the logs should now be sent to ELK.
```

Comment:
I think we always say "Administration Server" in the doc @rosemarymarano

Reply:
ELK -> Elasticsearch

Reply:
Yes, to both comments.

Reply:
I changed to "administration server" and left as lower case